Packs/AzureWAF/ReleaseNotes/1_2_11.md

```diff
@@ -0,0 +1,16 @@
+#### Modeling Rules
+
+##### Azure WAF Modeling Rule
+
+- Updated the Azure WAF Modeling Rule to support Azure WAF log types AGWAccessLogs and AGWFirewallLogs.
```

Comment:
Updated the Azure WAF modeling rule to support the AGWAccessLogs and AGWFirewallLogs Azure WAF log types.